index.html

```diff
@@ -305,23 +305,20 @@ <h3>
           "RESOURCE-TIMING-2#dom-performanceresourcetiming">PerformanceResourceTiming</dfn></code>
           interface:
         </p>
-        <ul data-dfn-for="PerformanceResourceTiming">
-          <li>The <dfn>initiatorType</dfn> attribute MUST return the
-          {{DOMString}} "<code>navigation</code>".
+        <ul>
+          <li>The `workerStart` getter steps are to perform the following steps:
+            <ol>
+              <li>Let |workerTiming| be |this|'s [=PerformanceNavigationTiming/service worker timing=].
+              <li>If |workerTiming| is null, then return |this|'s prototype's `workerStart`.
```

Comment:
Are there cases where |workerTiming| is null, but resource timing's workerStart would have a meaningful value?

Reply:
Yes, if the navigation reused an existing service worker (e.g. an iframe).